tsconfig.json

```diff
@@ -27,8 +27,8 @@
     // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
 
     /* Additional Checks */
-    "noUnusedLocals": true                /* Report errors on unused locals. */
-    // "noUnusedParameters": true,            /* Report errors on unused parameters. */
+    "noUnusedLocals": false,                /* Report errors on unused locals. */

```

Comment:
Please do not change this file in your PR. You may make these changes locally to test, but we shouldn't be changing the settings that were decided upon on the clean branch

Reply:
Again, this was just a case of changing whatever git was complaining about when I tried to merge the two branches. I'm not sure if I'm doing something wrong, or if I should just create the PR without the merge going through...anyway, sorry about the mess